src/core/videosource.cpp

```diff
@@ -252,7 +252,6 @@ FFMS_VideoSource::FFMS_VideoSource(const char *SourceFile, FFMS_Index &Index, in
 
         DecodeFrame = av_frame_alloc();
         LastDecodedFrame = av_frame_alloc();
-        StashedPacket = av_packet_alloc();
 
         if (!DecodeFrame || !LastDecodedFrame || !StashedPacket)
```

Comment:
StashedPacket is still checked here fwiw.

Reply:
Thanks, fixed.